src/lib/styles/themes/dark.scss

```diff
@@ -146,6 +146,7 @@
   // Sidebar
   --sidebar-button-background: var(--cp-dark-1000-20);
   --sidebar-button-background-hover: var(--cp-dark-900);
+  --sidebar-icon: var(--text-color);
```

Comment:
do we need this new global variable?

Reply:
We don't need it, I am trying to keep everything meaningfully in sync with Figma. I can use --text-color directly as well.

Reply:
Ah ok I understand. I tend to not add global variable that may not be needed but if it fits Figma sure that's fine for me.